dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -5113,17 +5113,24 @@ ReturnedColumn* buildAggregateColumn(Item* item, gp_walk_info& gwi)
 
             // @bug 3603. for cases like max(rand()). try to build function first.
             if (!rc)
+            {
               rc = buildFunctionColumn(ifp, gwi, gwi.fatalParseError);
-
+            }
+#if 0
```

Comment:
Sorry but I missed the reason why we need this commented out block here.

Reply:
removed the commented code